package.json

```diff
@@ -51,9 +51,9 @@
     "@babel/preset-react": "^7.16.0",
     "@babel/preset-typescript": "^7.16.0",
     "@jest/types": "^27.2.5",
-    "@remix-run/node": "^1.10.0",
-    "@remix-run/react": "^1.10.0",
-    "@remix-run/server-runtime": "^1.10.0",
+    "@remix-run/node": "^1.13.0",
+    "@remix-run/react": "^1.13.0",
+    "@remix-run/server-runtime": "^1.13.0",
```

Comment:
These should be the same version as the peer dependencies.

Reply:
I've made this change but FYI the installed version in the lock is still 1.13.0.